packages/runtime/container-runtime/src/summary/summaryGenerator.ts

```diff
@@ -146,23 +146,22 @@ export class SummarizeResultBuilder {
 	public fail(
 		message: string,
 		error: any,
-		stage: SummaryStage = "unknown",
+		broadcastFailedResult: IBaseSummaryFailureResult = { stage: "unknown" },
```

Comment:
I'm not sure the defaults make sense here and below.  Feels like it would be most clear to avoid having any default values here at all.  I haven't looked at callers to see who was previously not passing `stage`.

Anyway, under default values, the submitted failure data was `{stage:"unknown"}` before but now would be `undefined`. Right? Good spot to hit with UTs, maybe you did.

(and probably good to line up the order of the 3 phases in the params and the code, for lower cognitive load when reading)

Reply:
I simplified the whole logic, so this change is mostly removed.